### PR TITLE
VACMS-8519: Adds test coverage for file upload.

### LINKED ISSUE #8519: Add test coverage for file upload
## User Story or Problem Statement

This week we had two failures simultaneously related to file upload:
1) due to dropzone 2.6 adding user perms that did not exist previously
2) due to unrelated custom code not scoped appropriately and failing file form submission.

Neither of these were caught by our current testing stack and unfortunately they both snuck through until users reported an issue.

As an engineer for the CMS I would like an automated test that covers file upload to validate a file can be added and that it actually saves and makes its way into the system (not just that weird temp file thing that drupal does until node save).

## Implementation suggestions:
* Probably events is the easiest node content type to check this on, rather than just testing it at the add media level.
* Only needs to use the file select button and does not need to check drag and drop into dropzone.  (mainly because I have no idea how we could do the drag and drop test... and the herculean effort is probably not worth it)

## Acceptance Criteria
- [ ] When file upload is broken, our testing pipeline reveals it.
     - [ ] Covers process within node/add to test full upload and save cycle (not just temp storage that drupal does before node save)

### CMS Team
Please check the team(s) that will do this work. 

- [ ] `CMS Program`
- [x] `Platform CMS Team`
- [ ] `Sitewide CMS Team ` (leave Sitewide unchecked and check the specific team instead)
  - [ ] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product support`
  - [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

Closes #8519.

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As user _uid_ with _user_role_
1. Do this
   - [ ] Validate that
2. Then
   - [ ] Validate that
3. Then validate Acceptance Criteria from issue
   - [ ] a
   - [ ] b
   - [ ] c

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Platform CMS Team`
- [x] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: tests/behavioral/cypress/integration/common/i_create_a_node.js [hunk 1]
```diff
@@ -11,6 +11,34 @@ const creators = {
...
+    cy.findAllByLabelText('State').select('Alabama', { force: true });
+    cy.findAllByLabelText('Section').select('VACO', { force: true });
+    cy.scrollToSelector('#edit-field-media-open-button');
+    cy.get('#edit-field-media-open-button').click({ force: true });
+    cy.get('.dropzone', {
+      timeout: 10000,
+    });
+    cy.get('.dropzone').attachFile('images/polygon_image.png', {
+      subjectType: 'drag-n-drop',
+    });
```
Comment (ndouglas) [inline tests/behavioral/cypress/integration/common/i_create_a_node.js:32]: This uses the drag and drop functionality 😃

**Comment by ndouglas** [comment]
@swirtSJW since you asked 😉

File: tests/behavioral/cypress/integration/common/i_create_a_node.js (hunk 1 above)
Comment (swirtSJW) [reply tests/behavioral/cypress/integration/common/i_create_a_node.js:32]: Ooooh so many bonus points earned!!!

File: tests/behavioral/cypress/integration/common/i_should_see.js [hunk 2]
```diff
@@ -12,3 +12,12 @@ Then('I should not see an element with the xpath {string}', (expression) => cy.x
 
 Then('I should see xpath {string}', (expression) => cy.xpath(expression).should('exist'));
 Then('I should not see xpath {string}', (expression) => cy.xpath(expression).should('not.exist'));
+
```
Comment (swirtSJW) [inline tests/behavioral/cypress/integration/common/i_should_see.js:15]: slick behat like addition.

**Review by swirtSJW — APPROVED** [review]
Incredible work Nate.  Thank you.

**Pull request merged**